### PR TITLE
feat(DDI-998): Added example for linecount prop

### PR DESCRIPTION
**Pull request by Radha-Alberta**
Added example for linecount prop

### CODE REVIEW

File: libs/web-components/src/components/skeleton/Skeleton.svelte [hunk 1]
```diff
@@ -6,6 +6,7 @@
   import { calculateMargin } from "../../common/styling";
   import { onMount } from "svelte";
   import { typeValidator } from "../../common/utils";
+  import { to_number } from "svelte/internal";
```
Comment (chrisolsen) [inline libs/web-components/src/components/skeleton/Skeleton.svelte:9]: Probably shouldn't be using internal methods. Use `Number.parseInt` instead

Comment (Radha-Alberta) [reply libs/web-components/src/components/skeleton/Skeleton.svelte]: Done

**Review by ArakTaiRoth — APPROVED** [review]

**Pull request merged**